apollo-router/src/executable.rs

```diff
@@ -69,7 +69,7 @@ pub struct Opt {
 
     /// The endpoint polled to fetch the latest supergraph schema.
     #[clap(long, env)]
-    apollo_uplink_url: Option<Url>,
+    apollo_uplink_endpoints: Option<Url>,
```

Comment:
I assume this actually affects the env var used via the `env` on the previous line?

Reply:
It does!